hcloud/client.go

```diff
@@ -55,6 +55,7 @@ type Client struct {
 	FloatingIP FloatingIPClient
 	Server     ServerClient
 	SSHKey     SSHKeyClient
+	ServerType ServerTypeClient
```

Comment:
I try to keep the list sorted alphabetically, so please move `ServerType` below `Server`.